src/commonMain/kotlin/exchange.dydx.abacus/functional/vault/VaultDepositWithdrawForm.kt

```diff
@@ -0,0 +1,274 @@
+package exchange.dydx.abacus.functional.vault
+
+import kotlinx.serialization.Serializable
+import kotlin.js.JsExport
+
+@JsExport
+@Serializable
+data class FormData(
```

Comment:
Can we use prefix those data classes with "Vault" (i.e., VaultFormData, VaultTransactionAction, etc)?  This helps minimizing name-conflicts.

Reply:
oh yeah that's a good idea. Unfortunate that they're all globally scoped.